Sources/Transifex/Core.swift

```diff
@@ -190,12 +190,25 @@ class NativeCore : TranslationProvider {
                                         arguments: arguments)
     }
     
-    /// TranslationProvider protocol method used by Swizler class.
+    /// TranslationProvider protocol method used by Swizzler class.
     func translate(sourceString: String, params: [String : Any]) -> String {
-        return translate(sourceString: sourceString,
-                         localeCode: nil,
-                         params: params,
-                         context: nil)
+        /// If this call call originates from a `localizedStringWithFormat` swizzled method, it will
+        /// contain the extra arguments. In that case he first argument of those methods (format) would
+        /// have already been resolved by a `NSLocalizedString()` call, so we don't need to
+        /// perform a second fetch on the cache, we can proceed by directly rendering the strict and let the
```

Comment:
If I remember, and understand, correctly, it's not just that we don't need to but rather that we shouldn't. Before this change, the behavior wasn't correct, correct? 🙂

Reply:
Correct, I will adjust the comment there.